web/src/views/DandisetLandingView/DandisetLandingView.vue

```diff
@@ -117,8 +117,8 @@
                 <template v-if="meta.contributors">
                   <v-subheader>Contributors</v-subheader>
                   <v-list-item
-                    v-for="(item, i) in meta.contributors"
-                    :key="i"
+                    v-for="item in meta.contributors"
+                    :key="item.orcid || `${item.name}-${item.roles}`"
```

Comment:
If a given `item` will always consistently yield the same unique result from this, it will work. I'm not familiar with the schema of `meta.contributors`, is `${item.name}-${item.roles}` alone not guaranteed to yield a unique result?

Reply:
I'd say it's at least unlikely to yield the same results. Obviously using the `orcid` is the best case scenario, as those are guaranteed to be unique. However if that isn't supplied, I think it's unlikely that two people with the exact same name and list of roles will be supplied.

Reply:
Sounds reasonable.